tests/tests.spec.js

```diff
@@ -0,0 +1,69 @@
+const { test, expect } = require('@playwright/test');
+const { LoginPage } = require('../pages/loginPage');
+const { ProductPage } = require('../pages/productPage');
+const { CartPage } = require('../pages/cartPage');
+const { standardUser } = require('../test_data/users.json');
+const { CheckoutPage } = require('../pages/checkoutPage');
+const { ConfirmationPage } = require('../pages/confirmationPage');
+const { CompletionPage } = require('../pages/completionPage');
+
+const sauceLabsBikeLight = "Sauce Labs Bike Light";
+const sauceLabsBackpack = "Sauce Labs Backpack";
+const sauceLabsFleeceJacket = "Sauce Labs Fleece Jacket";
+
+test('Make an order with a product', async ({ page }) => {
+  const loginPage = new LoginPage(page);
+  const productPage = new ProductPage(page);
+  const cartPage = new CartPage(page);
+  const checkoutPage = new CheckoutPage(page);
+  const confirmationPage = new ConfirmationPage(page);
+  const completionPage = new CompletionPage(page);
+
+
+  await loginPage.goto();
```

Comment:
I would put this steps to [beforeEach](https://playwright.dev/docs/api/class-test#test-before-each).